runner/src/client/sass/application.scss

```diff
@@ -4,6 +4,20 @@
 @import "modal-dialog";
 @import "upload-dialog";
 
+@mixin govuk-global-styles {
+  a {
+    @extend %govuk-link;
+  }
+
+  p {
+    @extend %govuk-body-m;
+  }
+}
+
+@include govuk-global-styles;
+
+$govuk-global-styles: true;
```

Comment:
all we need is `$govuk-global-styles: true;` here, however you may need to move this line before we import govuk-frontend styles. 

At compile time, since govuk-global-styles is true, when we import and compile govuk/all, it will use our declaration of `$govuk-global-styles: true;` rather than the default value in govuk-frontend.

Reply:
I think the behaviour changes slightly when running the code locally, i've removed the additional code